apps/hip/templates/hip/home_page.html

```diff
@@ -7,6 +7,14 @@
 
 {% block content %}
   <div class="homepage-padding-hip">
+    {% if page.title %}
+      <section class="pb-4 px-2">
+        <h3 class="is-size-3"><strong>{{ page.title }}</strong></h3>
```

Comment:
```suggestion
        <h2 class="is-size-2"><strong>{{ page.title }}</strong></h2>
```
I think this might be the more appropriate size. Let me know if you agree and we accept the suggestion.

Reply:
Since it's the page title, it seems like it should have the highest priority of headings. Since it looks like we use h2 for titles on other pages, I support this being an h2.